orchestra/migrations/0053_worker_slack_id.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-04-29 03:55
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('orchestra', '0052_auto_20160428_2142'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='worker',
+            name='slack_id',
```

Comment:
`slack_user_id` instead? the slash command returns a field called `user_id` so it might be better to be consistent with this (and also there are other types of ids like channels)